apps/reflect/src/types/index.ts

```diff
@@ -41,8 +41,14 @@ interface IBotActionActivity {
   commandAuthor: string | null;
 }
 
+interface ITwitchBotConfig {
+  configName: any;

```

Comment:
The types of these properties are any because the react-hook-form setValue function didn't accept it when I tried to give them an appropriate type.